crates/holochain_state/src/query/live_record.rs

```diff
@@ -28,18 +27,14 @@ impl Query for GetLiveRecordQuery {
         SELECT Action.blob AS action_blob
         FROM DhtOp
         JOIN Action On DhtOp.action_hash = Action.hash
-        WHERE DhtOp.type IN (:create_type, :delete_type, :update_type)
```

Comment:
Filtered by op type, which didn't include `RegisterAddLink`. I could include that op type here, but I don't know why the op type is filtered here to begin with and think this could possibly address other failed lookups as well.